src/map/lapack2flash/FLASH_getrf.c

```diff
@@ -36,28 +37,46 @@
                                int* buff_p,                             \
                                int* info )
 
-// Note that p should be set zero.
 #define LAPACK_getrf_body(prefix)                               \
   FLA_Datatype datatype = PREFIX2FLAME_DATATYPE(prefix);        \
-  FLA_Obj      A, p;                                            \
+  FLA_Obj      A, p, AH, pH, LH;                                \
   int          min_m_n    = min( *m, *n );                      \
   FLA_Error    e_val;                                           \
   FLA_Error    init_result;                                     \
+  dim_t        blocksize = FLASH_get_preferred_blocksize();     \
                                                                 \
   FLA_Init_safe( &init_result );                                \
                                                                 \
   FLA_Obj_create_without_buffer( datatype, *m, *n, &A );        \
   FLA_Obj_attach_buffer( buff_A, 1, *ldim_A, &A );              \
                                                                 \
-  FLA_Obj_create_without_buffer( FLA_INT, min_m_n, 1, &p );     \
-  FLA_Obj_attach_buffer( buff_p, 1, min_m_n, &p );              \
-  FLA_Set( FLA_ZERO, p );                                       \
+  if ( *m == *n )                                               \
```

Comment:
FLASH_LU_incpiv and FLASH_LU_piv both died when m != n in the netlib test. (./LIN/xlintsts < stest.in)

Making this use the FLA_LU_piv if m != n made the test get farther. However there seems to be another issue making the test fail when it gets to m == n == 50.